aiplatform/snippets/list_models.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+package snippets
+
+// [START aiplatform_list_models]
+
+import (
+	"context"
+	"fmt"
+
+	aiplatform "cloud.google.com/go/aiplatform/apiv1"
+	"cloud.google.com/go/aiplatform/apiv1/aiplatformpb"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+// listAllModels lists all of the Vertex AI models in a project for
+// a specific region.
+func listAllModels(projectID, region string) error {
```

Comment:
optional nit: I find the "multiple variables of the same type" style of declaration less readable.

Reply:
Changed (and good point).